### ISSUE TITLE
default interactive option can be simpler

### ISSUE THREAD

**Issue by hamidzr** [StreisandEffect/streisand#1336]
What do you think of the following? wouldn't it be simpler for the novice users if the interactive installer's default choice was to select "standard" installation vs customized installation? 

> -Press enter to customize your installation: " confirm

```
diff --git a/streisand b/streisand
index a009fcb..4c2379e 100755
--- a/streisand
+++ b/streisand
@@ -69,10 +69,10 @@ function customize() {
 Do you wish to customize which services Streisand will install?
 By saying 'no' Streisand will use the settings configured in $SITE_VARS
 
-Press enter to customize your installation: " confirm
+Do you want to customize your installation? (yes/no) [no]: " confirm
   case "$confirm" in
-    no) echo; echo "Installing Streisand services specified in $SITE_VARS";;
-     *) echo; echo "Confirmed. Customizing Streisand services.";
+      *) echo; echo "Installing Streisand services specified in $SITE_VARS";;
+    yes) echo; echo "Confirmed. Customizing Streisand services.";
          # NOTE(@cpu): We don't pass the other `--extra-vars` here because the
          # customize `vars_prompt` will only happen if the vars aren't already
          # set. If you pass the site/defaults in no prompting will happen.
```

**Comment by nopdotcom** [comment]
Maybe the installer should say "Here's what I'm planning to do; would you like to customize it?"

```
* Here's what will be installed:

  o  OpenVPN
  o  Stunnel (for OpenVPN)
  o  Shadowsocks
  o  An ssh forwarding account
  o  Tinyproxy
  o  Wireguard

* Here's what won't be installed:

  o  OpenConnect
  o  Tor

* Number of client certificates: 5

* The SSH private key to use is: ~/.ssh/id_rsa

If you'd like to customize this, type yes: 
```

Does that make sense?

**Comment by nopdotcom** [comment]
We could just special-case the "no site.yml configured" situation; that way, we wouldn't have to write all the "parse current configuration" code. Presumably people who have customized it once already sorta know what's going on.

**Comment by hamidzr** [comment]
I agree it would be even better to show what the installer is going to do.
just to keep it simple (with the minimum amount of change) is to decide whether the default selection should be to customize or not

**Comment by cpu** [comment]
> Just to keep it simple (with the minimum amount of change) we need to see which would be better for your target audience:
> 
> 1. default action of customizing the installation1
> 2. accepting the standard configuration

We were doing 2. by default before https://github.com/StreisandEffect/streisand/commit/17ff1b0e8cad508c8ebede3de710b26c058df68d I'm hesitant to switch back. I don't think the default "everything enabled" configuration is the best choice for most users.

**Comment by hamidzr** [comment]
Thanks for your input, it makes sense. The first time I was using the installer it wasn't clear what I was customizing when I got prompted for it and I thought of as customizing in a sense that you customize a lot of linux-based services. feel free to close the issue now.

**Comment by cpu** [comment]
I'm open to using this issue to improve the language in the installer to make it clearer what the customization is for :-)

**Comment by hamidzr** [comment]
the format that @nopdotcom  suggested seems very good https://github.com/StreisandEffect/streisand/issues/1336#issuecomment-390046267. But maybe change the default answer to yes and prompt user using the standard selection.

**Comment by Frichetten** [comment]
Not to jump in out of nowhere but perhaps there should be some guidance as to what type of VPN the user would want? For example:

```
I'm going to install OpenVPN because it's the most widely supported VPN client.
I can also install the following:

  o  Stunnel (for OpenVPN)
  o  Shadowsocks
  o  An ssh forwarding account
  o  Tinyproxy
  o  Wireguard

If you'd like to customize this, type yes:
```
Obviously you could offer whatever type of VPN you want to the user (I used OpenVPN as an example). And I think @cpu is right, defaulting to installing all of the options isn't really necessary. Streisand doesn't really inform the user of what may be a good option (You could argue that isn't Streisand's place to teach the user about different VPN tech).

What would you guys think about defaulting to a specific VPN type, and then offering the user the chance to pick a different one?/
